### ISSUE TITLE
Strong Name Signing

### ISSUE THREAD

**Issue by michael-damatov** [Kinnara/ModernWpf#242]
Please sign the package assemblies with a strong name key. This would eliminate the following warning:

```
warning CS8002: Referenced assembly 'ModernWpf.Controls, Version=0.9.2.0, Culture=neutral, PublicKeyToken=null' does not have a strong name.
```

when the package is used by a signed assembly.

**Comment by ShankarBUS** [comment]
But doesn't code signing require money?

This project is opensource. You can't expect the author to spend their own money on this project.

Is it that necessary BTW?

**Comment by michael-damatov** [comment]
No, it doesn't require any money (this is **not** a code signing certificate issued by a CA). A key can easily be created by the VS.

**Comment by alex-lapuka** [comment]
@Kinnara  Are there any updates on that issue? It seems like the latest release is still without the strong name. That is preventing that NuGet package from using at any strong-named app.

**Comment by Kinnara** [comment]
I rechecked this against the current 1.x line. The built `ModernWpf` assembly is still unsigned (`PublicKeyToken=null`), and there is no signing key or signing configuration in the repository.

Strong-name signing itself does not require a paid certificate, but adding it now would change the public key token and therefore the shipped assembly identity. `1.0.0-preview.1` is the forward binary-compatibility baseline for 1.x, so that identity change requires a deliberate major-version migration rather than a compatible 1.x enhancement.

Closing this as not planned for 1.x. It can be reconsidered for a future major version together with key ownership, signing/release policy, and an assembly-binding migration plan.
